lobster/tools/codebeamer/codebeamer.py

```diff
@@ -525,17 +536,30 @@ def main():
 
     elif options.import_query:
         try:
-            query_id = int(options.import_query)
-        except ValueError:
-            ap.error("query-id must be an integer")
-        if query_id < 1:
-            ap.error("query-id must be a positive")
+            if isinstance(options.import_query, int):
+                query = int(options.import_query)
+                if query < 1:
+                    raise ValueError("query_string must be a positive integer"
```

Comment:
The error message is misleading; I will suggests that the issue is related to a query ID or something similar involving only integers.